test-result-summary-client/src/TabularView/TabularView.jsx

```diff
@@ -272,7 +274,6 @@ export default class TabularView extends Component {
         + '&baselineSdkResource=' + this.state.baselineSdkResource + '&baselineBuildServer=' + this.state.baselineBuildServer;
 
         window.history.replaceState(null, '', newPath);
-        window.location.reload(false);
```

Comment:
I suspect that this change might bring back the old bug: https://github.com/AdoptOpenJDK/openjdk-test-tools/issues/224 fixed in https://github.com/AdoptOpenJDK/openjdk-test-tools/pull/230.